### ISSUE TITLE
small adjustments for BLMP

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/owl-schema#60]
In the perspective of the BLMP building its UI by reading certain patterns in the ontology, it would be helpful if:

- `adm:OutlineType` was a subclass of `:Type`
- `adm:Product` was a subclass of `:Entity`

**Comment by berger-n** [comment]
I have the same issue about `:Taxonomy`

**Comment by xristy** [comment]
what should `:Taxonomy` be a subclass of?

**Comment by berger-n** [comment]
`:Type` I think

**Comment by xristy** [comment]
what is problem with being subclass of `:Entity`? `:Taxonomy` is intended to be the class of organizations of _tokens_ that are used to classify such as classifying `:Work`s but could be used to classify `:Place`s and so on.

**Comment by berger-n** [comment]
why not subclass of both `:Entity` and `:Type` ?

**Comment by berger-n** [comment]
(the thing is that `http://purl.bdrc.io/resource/EightMinorKagyuTraditions` `http://purl.bdrc.io/resource/EightPracticeLineages` `http://purl.bdrc.io/resource/FourMainKagyuTraditions` `http://purl.bdrc.io/resource/TraditionTaxonomy` seem to be possible choices for a `:Taxonomy`, in the same that :GenderFemale, :GenderMale, :GenderNotSpecified for :Gender which is a subclass of :Type)

**Comment by xristy** [comment]
originally the idea was that `:Entity`s would be the _top level_ ideas / concepts that that ontology _is about_ and the `:Type`s would be collections of _individuals_ that are the _objects_ of triples about `:Entity`s. It would an instance of cognitive dissonance for me to have `:Taxonomy` be a subclass of both `:Entity` and `:Type`.

What is the problem that having `:Taxonomy` as a subclass of `:Entity` creates for organizing BLMP functionality?

**Comment by berger-n** [comment]
every `:NamedIndividual` in the `.owl` file has a class that inherits from `:Type` (except these four which class is `:Taxonomy`) and this is the test used to choose which component to use to render the corresponding property...

(BTW for me the cosmic dissonance - lapsus for "cognitive dissonance" sorry - might be that `:Taxonomy` is the only `:Entity` that does not correspond to a root directory where `.ttl` files are stored in... ^^)

when loading the ontology the idea is to iterate over `:NamedIndividual`s so as we can know for each `:Type` which values it can take (and automatically use them as a popup menu, `:Gender` is a good example)

**Comment by berger-n** [comment]
can you think of an exemple resource that makes use of `:Taxonomy` ? I have some time ahead before my `grep` returns something... ah, spoken too fast, `G1TLMTGA000001` and `G1TLMTGB000001` seem to be the only two that do : both with `adm:place_TLM_taxonomy bdr:O1TLMXXX000011;` 

but I already agree that it may not be necessary at all from a technical point of view that `:Taxonomy` is a subclass of `:Type` 

as a matter of fact, I understand it would even be some kind of mistake to do so, and that there may be a `taxonomy` root directory in the future 

but what do of [these four](https://github.com/BuddhistDigitalResourceCenter/owl-schema/issues/60#issuecomment-353628263) then ? do they correspond to an exhaustive enumeration of something ? _why_ are they ?

**Comment by berger-n** [comment]
(sorry - did not refresh the page - seems that the discussion is not over yet... ^^)

**Comment by xristy** [comment]
There's a lot to say about the intended role of taxonomy in supporting BLMP and search. Are you familiar with "A Buddhist Digital Ontology"? I just shared it with you in case you have not yet explored the section  on taxonomy.

The two examples that you discovered `G1TLMTGA000001` and `G1TLMTGB000001` are monastic libraries that have define their own taxonomies for cataloging holdings and those taxonomies have not been encoded yet.

There is also the taxonomy of traditions that is reasonable to consider for use in subject classification and there is the use of taxonomy to organize `:Place` types.

Then there is the need to define and encode taxonomies for subject classification and genre classification in general.

Some of this is discussed in the narrative on the ontology that I just shared with you.
